starter-code/index.html

```diff
@@ -2,41 +2,89 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Spotify</title>
+  <title>SpotifyClone</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+  <header class="whitebackground">    
+    <nav class="nav whitebackground">
+      <div class="colLeft" >
+      <img src="images/spotify-logo.png" alt="logo-spotify">
+    
+      </div>
+
+      <div class="colRight">
+        <ul class="links">
+          <li class="links"> <a href="#0"> Premium </a> </li>
+          <li class="links"> <a href="#0"> Discover </a> </li>
+          <li class="links"> <a href="#0"> Help </a> </li>
+          <li class="links"> <a href="#0"> Download </a> </li>
+        </ul>
+    </div>
+  </nav>  
+</header>
+
+<div class ="clearfix musicForEveryOne">
+    <div class="musicForEveryOneTitle"> Music for everyone. </div>
+    <div class="musicForEveryOneText" >Spotify is now free on mobile, tablet and computer.
+    </br> Listen to the right music, wherever you are.
+    </div>
+</div>
+
+<div class = "wwhitebackground">
+  <h2>
+  What’s on Spotify?
+  </h2>
+
+    <div id="conteneur-avantages">
+      <div class="avantage"> 
+        <img class="icon" src="images/music-icon.png" alt="icon-loud-speaker">
+        <h3> Millions of Songs </h3>
+        <p>
+        There are millions of
+        songs on Spotify
+        </p>  
+      </div>
+
+      <div class="avantage"> 
+        <img class="icon" src="images/high-quality-icon.png" alt="icon-music-waves">
+        <h3>HD Music</h3>
+        <p>
+        Listen to music as if you
+        were listening live
+        </p>  
+      </div>
+
+      <div class="avantage">   
+        <img class="icon" src="images/devices-icon.png" alt="icon-phone-device">
+        <h3>Stream Everywhere</h3>
+        <p>
+        Stream music on your
+        smartphone, tablet or
+        computer
+        </p>  
+      </div>
+    </div>  
+</div>
+
+<div class="greenbackground">
```

Comment:
On this part, you should create a container that will host both of your content (the text and the image), it will be easier to move it like this : 

```html

    <div id="green-container">
      <div id="green-container-left">
        <h2>It’s as yeezy as Kanye West.</h2>
        <hr>
        <ul>
          <li>
            <h3>Search</h3>
            <p> Know what you want to listen to?
              <br>Just search and hit play. </p>
          </li>
          <li>
            <h3>Browse</h3>
            <p> Check out the latest charts,
              <br> brand new releases and great
              <br>playlists for right now. </p>
          </li>
          <li>
            <h3>Discover</h3>
            <p> Enjoy new music every Monday
              <br>with your own personal playlist.
              <br> Or sit back and enjoy Radio.
            </p>
          </li>
        </ul>
      </div>
      <div id="green-container-img">
        <img src="images/spotify-app.jpg">
      </div>
    </div>

```